homeassistant/scripts/check_config.py

```diff
@@ -368,9 +376,18 @@ def _comp_error(ex, domain, config):
                 platforms.append(p_validated)
                 continue
 
-            platform = loader.get_platform(hass, domain, p_name)
+            try:
+                p_integration = hass.loop.run_until_complete(
+                    loader.async_get_integration(hass, p_name))
+            except loader.IntegrationNotFound:
+                result.add_error(
+                    "Integration {} not found when trying to verify its {} "
+                    "platform.".format(p_name, domain))
+                continue
 
-            if platform is None:
+            try:
+                platform = p_integration.get_platform(domain)
+            except ImportError:
```

Comment:
We don’t seem to be unit testing for these ImportError blocks. Guess this should be a valid integration, with unimplemented platform. 

add_error is already tested in a couple of places, so should be ok